File: ide/idelsp/symbolresolve/testdata_zig/geometry.zig
pub fn area(radius: f64) f64 {
    return 3.14159 * radius * radius;
}

// Defined but never referenced as geometry.perimeter; resolves only
// through the definitions phase.
pub fn perimeter(radius: f64) f64 {
    return 2.0 * 3.14159 * radius;
}

pub const Shape = struct {
    size: f64,

    pub fn init(size: f64) Shape {
        return .{ .size = size };
    }
};

const Internal = struct {};

fn make_internal() Internal {
    return .{};
}

File: internal/ide/idelsp/symbolresolve/testdata_rs/geometry.rs
pub fn area(radius: f64) -> f64 {
    std::f64::consts::PI * radius * radius
}

// Defined but never referenced as geometry::perimeter; resolves only
// through the definitions phase.
pub fn perimeter(radius: f64) -> f64 {
    2.0 * std::f64::consts::PI * radius
}

pub struct Shape {
    size: f64,
}

impl Shape {
    pub fn new(size: f64) -> Shape {
        Shape { size }
    }
}

struct Internal;

fn make_internal() -> Internal {
    Internal
}